clients/python-wrapper/lakefs/branch.py

```diff
@@ -153,7 +153,7 @@ def uncommitted(self, max_amount: Optional[int], after: Optional[str] = None, pr
                                      **kwargs):
             yield Change(**diff.dict())
 
-    def import_data(self, commit_message: str, metadata: Optional[dict] = None) -> ImportManager:
+    def import_data(self, commit_message: Optional[str] = "", metadata: Optional[dict] = None) -> ImportManager:
```

Comment:
If it's optional it probably shouldn't be an empty string by default.

Reply:
It's optional in the SDK, it's a required param in the auto-generated code - empty string means no use default server commit message for import

Reply:
This comment make me suddenly uncertain: What happens if I pass an empty `commit_message`?  What happens if I pass `commit_message=None`?  This needs to be in the docstring: people who use the high-level SDK should _not_ need to refer to API documentation to extract such details.

Reply:
Passing commit_message=None will result in an error that the parameter is missing. Adding this to the docstring